Add pico-sdk@1.6.0-rc1

armandomontanez opened the pull request:
Add early testing release candidate for the Pico SDK.

bazel-io commented:
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (pico-sdk) have been updated in this PR. Please review the changes.

armandomontanez commented:
@kilograham This is adding the Pico SDK to Bazel's main package repository. I've added you as a maintainer, but I suspect you won't want to ever have to touch this so I've also added the Pigweed team as maintainers so we can manage these. Please let me know if you'd like me to adjust the maintainer list in any way.

armandomontanez commented:
Re: BcrValidationResult.FAILED: pico-sdk@1.6.0-rc1 is using an unstable source url

Is it possible to bypass for this PR? I'm ok with yanking this release after we get a full release of 1.6.0 cut if that's required.

Wyverald commented on modules/pico-sdk/1.6.0-rc1/presubmit.yml:
```diff
@@ -0,0 +1,14 @@
+      platform: ${{ platform }}
+      bazel: "7.x"
+      build_flags:
+        - "--platforms=@pico-sdk//bazel/platform:rp2040"
```
looks like you also need `--@rules_cc//cc/toolchains:experimental_enable_rule_based_toolchains` here
